Assets/Decontainment/Assembly/VirtualMachine.cs

```diff
@@ -164,6 +171,10 @@ public void Tick()
         OnTick?.Invoke();
     }
 
+    public void updateSharedReg(int registerNumber, int registerValue) {
```

Comment:
```suggestion
    public void UpdateSharedReg(int registerNumber, int registerValue)
    {
```
Keep the code style consistent